### PR TITLE
Issue 23 fix metadata

### PR DESCRIPTION
**Pull request by ryanprice**

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -9,40 +9,231 @@ import TokenContent from '@/components/TokenContent.vue'
...
+        meta: {
+          title: 'Cryptoz Cards - View Card - Detailed information about a Cryptoz Card',
+          metaTags: [
+                {
+                    name: 'description',
+                    content : 'While we do our best to make the Cryptoz Cards Experience as seamless as possible, this page offers answers and examples for most questions people have when getting started and even for the experienced player'
+                },
+                {
+                    property: 'og:url',
+                    content: 'https://cryptoz.cards/view',
```
Comment (ryanprice) [inline src/router/index.js:178]: Is there a way to make this dynamic, so the token id matches what is in the url ?  /view/234  for example ? I will research further

File: src/components/TokenContent.vue [hunk 2]
```diff
@@ -125,45 +125,28 @@
...
         Cryptoz.deployed().then(function(instance) {
           contract = instance;
-          return instance.exists.call(self.token_id);
-          //return instance.exists(self.token_id, {from: account});
+          return instance.getOwnedCard.call(self.token_id);
         }).then(function(res){
-          //console.log(res);
           //Does the token exist ?
-          if(res){
+          if(res[0].c[0] > 0){
```
Comment (jjjhill) [inline src/components/TokenContent.vue:131]: best if if statements are human readable (ie. variable 😋)

File: src/router/index.js [hunk 3]
```diff
@@ -175,7 +175,7 @@ const router = new Router({
                 },
                 {
                     property: 'og:url',
-                    content: 'https://cryptoz.cards/view',
+                    content: 'https://cryptoz.cards/view/{$token_id}',
```
Comment (jjjhill) [inline src/router/index.js:178]: did u test this? doesnt it have to be \`like ${this}\` with backticks specifically

**Review by jjjhill — COMMENTED** [review]
looks good besides 2 questions

File: src/router/index.js (hunk 3 above)
Comment (ryanprice) [reply src/router/index.js:178]: I tried the backticks but it didn't sub in the value.. still trying to figure it out

File: src/components/TokenContent.vue (hunk 2 above)
Comment (ryanprice) [reply src/components/TokenContent.vue]: ok replaced now

File: src/router/index.js (hunk 1 above)
Comment (ryanprice) [reply src/router/index.js]: found a way

File: src/router/index.js [hunk 4]
```diff
@@ -222,6 +223,12 @@ router.beforeEach((to, from, next) => {
     const tag = document.createElement('meta');
 
     Object.keys(tagDef).forEach(key => {
```
Comment (jjjhill) [inline src/router/index.js:225]: theres a nice short way to do this:
for (var key in tagDef) {

Comment (jjjhill) [reply src/router/index.js:225]: no need to change tho!

**Review by jjjhill — APPROVED** [review]
looks good!

**Comment by ryanprice** [comment]
ok guys going to merge, if you notice something, let me know !

**Pull request merged**